### PR TITLE
bendavey

### PR DESCRIPTION
**Pull request by bendavey**
Completed task 5.2P: Just prettying the place up a bit

### CODE REVIEW

**Review by Timkin — CHANGES_REQUESTED** [review]
Ben, you'll need to make some changes to your contribution. Another pull request was submitted prior to yours, but approved after you submitted yours (due to changes required... I'm doing that to each submission, to get you used to working with PRs and reviews). Could you please update your branch from the master again, and adjust your files to make sure everything works (when I tested auto-merging your files into the new robocity, I got several errors in the robocity.wbt file... it might need a manual merge). Let me know if you need help or guidance on this.

**Comment by bendavey** [comment]
I think I covered the conflicts (and the objects in Robocity, since John and I build in the same zone)

It looks like we're all using slightly different versions of Webots.

File: worlds/robocity.wbt
Comment (Timkin) [inline worlds/robocity.wbt]: road_2 name already exists - please update to reflect the ID change you made

**Review by Timkin — CHANGES_REQUESTED** [review]
One more error to deal with (commented). Once that's fixed, your changes can be merged. Thus, your last action is to take a new screenshot of the city (from your preferred location) using the internal camera tool, add this to the media folder (with appropriate version name, not deleting any past images) and update the README file to show your new image. Well done!

**Comment by bendavey** [comment]
Everything should be ready to merge!

**Comment by Timkin** [comment]
Thanks for dealing with this so promptly. Nice screenshot too!

**Pull request merged**
